pyflakes/checker.py

```diff
@@ -697,6 +697,11 @@ def ignore(self, node):
     # additional node types
     COMPREHENSION = KEYWORD = FORMATTEDVALUE = handleChildren
 
+    def ASSERT(self, node):
+        if isinstance(node.test, ast.Tuple):
```

Comment:
Nice catch.  I've excluded that bizarre case.

You may have seen that I also posted on https://bugs.launchpad.net/pyflakes/+bug/848467 , linking to an expanded solution which trials detecting static asserts (which is where `assert ()`, should be reported, if desirable)